zds/tutorialv2/tests/tests_views.py

```diff
@@ -5287,6 +5287,11 @@ def test_beta_article_closed_when_published(self):
             follow=False)
         self.assertEqual(result.status_code, 302)
 
+        # Check that the staff haven't a notification for its reservation and the private topic should be read.
```

Comment:
Non.

* `the staff haven't a notification` est incompréhensible, soit il manque un verbe (`the staff haven't baked a cake`) soit le pluriel est faux et au singulier ça donnerait `the staff user has no cake` ou `the staff user doesn't have a cake`
* `for its reservation` le staff n'est pas un objet
* `the private topic should be read` ici `should` n'est pas cohérent avec le verbe de modalité précédent